packages/edit-site/src/components/sidebar-navigation-screen-page/index.js

```diff
@@ -90,7 +90,9 @@ export default function SidebarNavigationScreenPage() {
 					className="edit-site-sidebar-navigation-screen__page-link"
 					href={ record.link }
 				>
-					{ record.link.replace( /^(https?:\/\/)?/, '' ) }
+					{ decodeURI(
+						record.link.replace( /^(https?:\/\/)?/, '' )
+					) }
```

Comment:
I believe the core uses the `safeDecodeURIComponent` from the `url` [package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-url/#safedecodeuricomponent), in other places for similar fixes. We can also replace the regex here with `filterURLForDisplay`.

```js
{ filterURLForDisplay( safeDecodeURIComponent( record.link ) ) }
 ```

Reply:
Thank you, I did not know these existed.